cats/__init__.py

```diff
@@ -193,6 +194,13 @@ def positive_integer(string):
         action="store_true",
         help="Disable all terminal output colouring (alias to --no-colour)",
     )
+    parser.add_argument(
+        "--plot",
+        help="Create a plot of the forecast and optimised plan for the job."
+        "This needs matplotlib to be installed, e.g. install with"
```

Comment:
```suggestion
        help="Create a plot of the forecast and optimised plan for the job. "
        "This needs matplotlib to be installed, e.g. install with "
```

to avoid lack of spaces in the output e.g. "job.This"